bin/run1_cpu_generate_rf.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+"""
+Generates Random Forest predictor for Run 1 CPU times, based on the iPython
+ML notebook by Phil Marshall
+https://github.com/drphilmarshall/StatisticalMethods/blob/master/examples/SDSScatalog/Quasars.ipynb
+and updates by Humna Awan and Tom McClintock
+https://github.com/DarkEnergyScienceCollaboration/Twinkles/blob/master/examples/notebooks/Run1CPUTimes.ipynb
+
+@author: sethdigel
+"""
+from __future__ import print_function
```

Comment:
Belated thanks, Jim.  I somehow lost track of this.  I've made the changes now that you recommended.